lib/repl.js

```diff
@@ -1126,7 +1126,10 @@ function isRecoverableError(e, self) {
       self._inTemplateLiteral = true;
       return true;
     }
-    return /^(Unexpected end of input|Unexpected token)/.test(message);
+
+    return message.startsWith('Unexpected end of input') ||
+      message.startsWith('Unexpected token') ||
+      message.startsWith('missing ) after argument list');
   }
```

Comment:
Oh lol. Sorry, missed that entirely ;)
On Nov 17, 2015 10:28 PM, "thefourtheye" <notifications@github.com> wrote:

> In lib/repl.js
> <https://github.com/nodejs/node/pull/3823#discussion_r45164076>:
>
> > @@ -1126,7 +1126,10 @@ function isRecoverableError(e, self) {
> >        self._inTemplateLiteral = true;
> >        return true;
> >      }
> > -    return /^(Unexpected end of input|Unexpected token)/.test(message);
> > +
> > +    return message.startsWith('Unexpected end of input') ||
> > +      message.startsWith('Unexpected token') ||
> > +      message.startsWith('missing ) after argument list');
> >    }
>
> Yup. We had discussed that in one of the old comments :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/3823/files#r45164076>.
>